templates/toil-infra-essentials.yaml

```diff
@@ -0,0 +1,63 @@
+AWSTemplateFormatVersion: 2010-09-09
+Resources:
+  AWSKmsInfraKey:
+    Type: "AWS::KMS::Key"
+    Properties:
+      Description: !Join
+        - '-'
+        - - !Ref AWS::StackName
+          - "InfraKey"
+      KeyPolicy:
+        Version: "2012-10-17"
+        Statement:
+          -
+            Sid: "Allow administration of the key"
+            Effect: "Allow"
+            Principal:
+              AWS:
+                - !Join
+                  - ''
+                  - - 'arn:aws:iam::'
+                    - !Ref AWS::AccountId
+                    - ':root'
+                - !ImportValue us-east-1-bootstrap-TravisUserArn
+                - !ImportValue us-east-1-bootstrap-CfServiceRoleArn
+                - !ImportValue us-east-1-rna-seq-reprocessing-instance-role-v001-ToilClusterRole
```

Comment:
this section grants permission to administer the kms key.  i think you just need to give toil cluster permission to use the key which is done below.